backend/prisma/schema.prisma

```diff
@@ -66,16 +66,11 @@ model User {
 
   email       String @unique
   password    String @map("password")
-  yearOfStudy Int    @map("year_of_study")
```

Comment:
If you are getting rid of yearOfStudy please add it to the enrollment model

Reply:
And make sure to update the zodschemas as well for enrollment if there is any

Reply:
fixed